web/src/components/roadmap/RoadmapSidebarCard.tsx

```diff
@@ -12,6 +12,7 @@ import LinearProgress, { linearProgressClasses } from "@mui/material/LinearProgr
 
 type Props = {
   card: SidebarCardContent;
+  cTAOnClickMap: Record<string, () => void>;
```

Comment:
should this be an optional prop? that way you don't have to add an empty map to all the tests. and your check on line 192 looks like you expected it to be optional

Reply:
also, for proper camel case naming, can we do `ctaOnClickMap`